notices.json

```diff
@@ -1,5 +1,24 @@
 [
     {
+    "id": "use_case_survey",
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "instanceType": "all",
+      "repeatable": "false",
+      "displayDate": "> 2022-08-01T00:00:00Z <= 2022-09-30T00:00:00Z"
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "We want your feedback!",
+        "description": "We are looking for feedback on how we are doing in supporting your team's collaboration throughout different phases of the DevOps lifecycle. Please help us with a quick 5 minutes survey and a chance to win a set of AirPod Pros.",
```

Comment:
Propose: and a chance... > for a chance...

Reply:
Updated